ffi/examples/read-table/expression_test_results.txt

```diff
@@ -0,0 +1,78 @@
+And
```

Comment:
@nicklan @zachschuermann  How should I go about testing this? I  imagine `ffi/tests/something` is a sensible place to put this file. I need some way to pass this file to the executable, but `read_table` expects a table input, not this.  

Currently I have`test_kernel_expr` pointing directly to `./expression_test_results.txt`, but that makes it so that you can only run the executable from the `read-table` directory.

Reply:
what is `test_kernel_expr`? maybe a `c` file you forgot to add?

I think you could have two binaries compiled in the `read-table` example with adding something like:
```cmake
add_executable(test_expr test_expr.c)
target_compile_definitions(test_expr PUBLIC DEFINE_DEFAULT_ENGINE)
target_include_directories(test_expr PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../../../target/ffi-headers")
target_link_directories(test_expr PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../../../target/debug")
target_link_libraries(test_expr PUBLIC delta_kernel_ffi)
target_compile_options(test_expr PUBLIC)
```

in the `CMakeLists.txt`.

That's not ideal, even better would be a new `print_expr` example. There's not much (if anything) you rely on from `read_table.h`, so you could just copy over whatever you need into a new header. lmk if you need help with getting that set up

Reply:
Also note that I removed the expression test in the C code for now since that breaks builds. It currently Works On My Machine ™, but I'd like to make it part of CI if possible.

Reply:
Just saw your msg. I'll get a second binary setup 👍